guacamole-common/src/main/java/org/apache/guacamole/GuacamoleException.java

```diff
@@ -68,5 +68,15 @@ public GuacamoleException(Throwable cause) {
     public GuacamoleStatus getStatus() {
         return GuacamoleStatus.SERVER_ERROR;
     }
+
+    /**
+     * Returns the numeric HTTP status code associated with this exception.
```

Comment:
I think the semantics of the value returned here need to be more clearly defined. With `GuacamoleStatus`, for example, `getHttpStatusCode()` is defined as returning "the most applicable HTTP error code". In this case, we should be clear that the HTTP status code should be the nearest equivalent to the status represented by this exception.

Reply:
Reworded it a bit.